samples/CMakeSamplesTemplate.txt

```diff
@@ -29,6 +29,9 @@ get_filename_component(SAMPLE_DIR_NAME ${TARGET_DIR} NAME)
 
 set(CUDA_INSTALL_DIR /usr/local/cuda)
 
+# Make sure the shared library path is correct for samples
+set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
```

Comment:
This would prevent the user from using the released libraries with the samples.  You should be using `LD_LIBRARY_PATH` instead.  This is also mentioned in the installation guide when using the tar installation method.  Also if you moved the libraries to another directory you'd need `LD_LIBRARY_PATH` to fix the issue.  I'd suggest we document `LD_LIBRARY_PATH` instead.

Reply:
I took Eric Work's advice. `CMakeSamplesTemplate.txt` was reverted. Instead, the `Dockfile`s were modified by adding new environment variables. Please review again.